### PR TITLE
[ES] Update `People/Beatmap Nominators/Becoming a Beatmap Nominator`

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/People/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/es.md [hunk 1]
```diff
@@ -1,34 +1,29 @@
...
 
-- Han acumulado al menos 150 [kudosu](/wiki/Modding/Kudosu).
-- Proporcionan 3 de sus mods que creen que los calificarían para ser parte de los Beatmap Nominators.
-  - Los miembros anteriores del BN/NAT pueden enviar un mínimo de 2 mods para su solicitud.
+- Tener al menos 150 [kudosu](/wiki/Modding/Kudosu).
+- No haber infringido las [normas de la comunidad de osu!](/wiki/Rules) en los últimos 12 meses.
+- Tener la capacidad de reconocer beatmaps que deberían (y no deberían) ser nominados.
 
-### Expectativas de modding
+## Rasgos a demostrar en el modding
```
Comment (agatemosu) [inline wiki/People/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/es.md:17]:
```suggestion
## Cualidades a demostrar en el modding
```

File: wiki/People/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/es.md [hunk 2]
```diff
@@ -1,34 +1,29 @@
...
 
 *Véase también: [Errores de modding comunes](/wiki/Guides/Common_modding_mistakes) y [Guía de modding de osu!mania](/wiki/Guides/osu!mania_modding_guide)*
 
-Los mods enviados deben mostrar que un usuario es capaz de analizar y mejorar beatmaps hasta un nivel adecuado para los Beatmap Nominators.
+Los mods enviados deben mostrar que un usuario es capaz de reconocer los beatmaps que deben y no deben ser nominados, además de analizar y mejorar los beatmaps hasta un nivel adecuado para los Beatmap Nominators.
 
-Los siguientes son los rasgos de modding que se esperan de los Beatmap Nominators. La falta de mostrar cualquiera de estos rasgos puede resultar en una solicitud rechazada.
+Los siguientes son los rasgos que se esperan de los Beatmap Nominators en el modding:
```
Comment (agatemosu) [inline wiki/People/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/es.md:23]:
```suggestion
Los siguientes son los rasgos que se esperan en el modding dentro de los Beatmap Nominators:
```

File: wiki/People/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/es.md [hunk 3]
```diff
@@ -77,37 +100,20 @@ La *etapa grupal* dura como máximo hasta 14 días después de que el solicitant
...
+## Tiempos de espera
 
-## Notas
+Hay un tiempo de espera de 60 días para volver a aplicar en las siguientes ocasiones:
 
-[^good-terms]: Un Beatmap Nominator que renunció en buenos términos será tratado con términos estándar después de 1 año.
-[^activity-terms]: Los solicitantes deberán proporcionar 3 mods en las solicitudes, en lugar de al menos 2.
+- Solicitudes fallidas
+- Ser retirado de los Beatmap Nominators
+- [Renunciar en términos estándar](/wiki/People/Nomination_Assessment_Team/Evaluations#standard-terms)
```
Comment (agatemosu) [inline wiki/People/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/es.md:119]:
```suggestion
- [Renunciar en buenos términos](/wiki/People/Nomination_Assessment_Team/Evaluations#standard-terms)
```

File: wiki/People/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/img/bn-application-flowchart-ES.png
Comment (agatemosu) [inline wiki/People/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/img/bn-application-flowchart-ES.png:1]: 😍

**Comment by UnknownOsu** [comment]
Se me coló la `a`😭

File: wiki/People/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/es.md [hunk 4]
```diff
@@ -1,34 +1,29 @@
...
-### Expectativas de modding
+## Cualidades a demostrar en el modding
 
 *Véase también: [Errores de modding comunes](/wiki/Guides/Common_modding_mistakes) y [Guía de modding de osu!mania](/wiki/Guides/osu!mania_modding_guide)*
 
-Los mods enviados deben mostrar que un usuario es capaz de analizar y mejorar beatmaps hasta un nivel adecuado para los Beatmap Nominators.
+Los mods enviados deben mostrar que un usuario es capaz de reconocer los beatmaps que deben y no deben ser nominados, además de analizar y mejorar los beatmaps hasta un nivel adecuado para los Beatmap Nominators.
 
-Los siguientes son los rasgos de modding que se esperan de los Beatmap Nominators. La falta de mostrar cualquiera de estos rasgos puede resultar en una solicitud rechazada.
+Lo siguiente son las cualidades que se esperan de los Beatmap Nominators en el modding:
```
Comment (agatemosu) [inline wiki/People/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/es.md:23]:
```suggestion
Estas son las cualidades que se esperan en el modding dentro de los Beatmap Nominators:
```

**Review by agatemosu — APPROVED** [review]

**Review by cl8n — APPROVED** [review]

**Pull request merged**